fcrepo-http-api/src/main/java/org/fcrepo/http/api/url/HttpApiResources.java

```diff
@@ -123,8 +123,6 @@ private void addNodeStatements(final FedoraResource resource, final UriInfo uriI
 
                 //RDF the serialization
                 final Resource formatRDF = createResource(REPOSITORY_NAMESPACE + key);
-
-                model.add(formatRDF, RDFS_LABEL, key);
                 model.add(format, dcFormat, formatRDF);
```

Comment:
That triple is about the "export" endpoint. Unless it's actually blocking @osmandin right now, I'm inclined to ignore it and wait 'til our later moves to genuine HTTP conneg to rip it out at that time.